README.md

```diff
@@ -318,6 +318,32 @@ Running Lidy's tests:
 go test
 ```
 
+Install Lidy in your project:
+
+```sh
+npm install "https://github.com/ditrit/lidy-js.git#main" --save
+```
+
+Generate parser according to your grammar in your project:
+```sh
+# Create a generate-parser.js in [YOUR_PROJECT]/scripts with this content: 
+import('lidy-js/parser/node_parse.js').then(({ preprocess }) => {
+  preprocess('src/lidy/[YOUR_GRAMMAR_NAME].yml');
+});
+
+# Add this line to your package.json script:
+"generate:parser": "node scripts/generate_parser.js"
+
+# Then place your grammar in src/lidy/[YOUR_GRAMMAR_NAME].yml, then run this commands:
```

Comment:
place the grammar at the good place should be the first step for me.

Reply:
done